libs/sdk-backend-base/src/cachingBackend/index.ts

```diff
@@ -224,6 +228,19 @@ class WithExecutionResultCaching extends DecoratedExecutionResult {
         return this.allData;
     };
 
+    public readForecastAll = (config: IForecastConfig): Promise<IForecastResult> => {
+        // TODO: enable forecasting caching size configuration
```

Comment:
Should we create a ticket for this? Or handle it right now?

Reply:
https://gooddata.atlassian.net/browse/F1-305